quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/services/QuarkusCompletions.java

```diff
@@ -88,26 +98,62 @@ private static void collectPropertyKeySuggestions(TextDocument document, Propert
 
 			CompletionItem item = new CompletionItem(property.getPropertyName());
 			item.setKind(CompletionItemKind.Property);
-			if (snippetsSupported) {
-				String defaultValue = property.getDefaultValue();
-				StringBuilder insertText = new StringBuilder();
-				insertText.append(property.getPropertyName());
-				insertText.append(' '); // TODO: this space should be configured in format settings
-				insertText.append('=');
-				insertText.append(' '); // TODO: this space should be configured in format settings
-				if (defaultValue != null) {
+
+			String defaultValue = property.getDefaultValue();
+			Collection<String> enums = getEnums(property);
+
+			StringBuilder insertText = new StringBuilder();
+			insertText.append(property.getPropertyName());
+			insertText.append(' '); // TODO: this space should be configured in format settings
+			insertText.append('=');
+			insertText.append(' '); // TODO: this space should be configured in format settings
+
+			if (enums != null) {
+				// Enumerations
+				if (snippetsSupported) {
+					SnippetsBuilder.choice(1, enums, insertText);
+				} else {
+

```

Comment:
Just double checking, was there supposed to be something here?

Reply:
Good catch. I fix it and add tests for snippetSupport or not.